docs/uart.rst

```diff
@@ -61,39 +58,45 @@ Functions
 
 .. method:: uart.any()
 
-   Return ``True`` if any characters waiting, else ``False``.
+    Return ``True`` if any characters waiting, else ``False``.
 
 .. method:: uart.read([nbytes])
 
-   Read characters.  If ``nbytes`` is specified then read at most that many
-   bytes.
+    Read characters.  If ``nbytes`` is specified then read at most that many
+    bytes.
+
+    .. note::
+
+        The timeout for all UART reads depends on the baudrate and is otherwise
+        not changeable via Python. The timeout, in milliseconds, is given by: 
+        ``microbit_uart_timeout_char = 13000 / baudrate + 1``
```

Comment:
Actually, could you also combine here the text from https://github.com/bbcmicrobit/micropython/pull/571 ?
We have quite a few notes/warnings in this document, and that note is only really relevant to the read method as well.